3rdparty/README.md

```
@@ -0,0 +1,341 @@
+The package will be installed by build\_wheel.py during virtual
+environment initialization prior to configuring the build with cmake.
+Include a comment indicating the intended usage of the package.
+
+[4]: https://gitlab-master.nvidia.com/ftp/tekit/-/blob/main/requirements-dev.txt
+[5]: https://gitlab-master.nvidia.com/ftp/tekit/-/blob/main/requirements-dev-windows.txt
+
+**Example:**
+
+`requirements-dev.txt`:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)